be/src/olap/rowset/segment_v2/inverted_index_compaction.cpp

```diff
@@ -80,19 +59,19 @@ Status compact_column(int32_t index_id, int src_segment_num, int dest_segment_nu
     for (auto* d : src_index_dirs) {
```

Comment:
If src_index_dirs and dst_index_dirs are not owned here, close() can also be moved to owner.